### PR TITLE
Add ONNX support for ConstantPadNd

### PR DESCRIPTION
**Pull request by houseroad**
Address the issue in ONNX: onnx/onnx/issues/21, https://github.com/onnx/onnx/issues/21

### CODE REVIEW

File: torch/nn/_functions/padding.py [hunk 1]
```diff
@@ -4,6 +4,17 @@
 class ConstantPadNd(Function):
 
     @staticmethod
+    def symbolic(g, input, pad, value=0):
+        dim = len(input.type().sizes())
+        paddings = []
+        for i, j in zip(pad[0::2], pad[1::2]):
```
Comment (apaszke) [inline torch/nn/_functions/padding.py]: Can you add a comment saying what's the expected ONNX format for padding sizes?

Comment (houseroad) [reply torch/nn/_functions/padding.py]: Sure.

Comment (dzhulgakov) [reply torch/nn/_functions/padding.py]: check that pad is divisible by 2? (zip just ignores stuff)

Comment (houseroad) [reply torch/nn/_functions/padding.py]: This is already checked by padding op in pytorch. So we won't have odd pad. I will add comments mentioning this. :-)

**Review by ezyang — APPROVED** [review]
Don't forget to add a test in onnx-pytorch.

**Pull request merged**